### ISSUE TITLE
Missing Alt text on images 

### ISSUE THREAD

**Issue by data-sync-user** [MozillaFoundation/foundation.mozilla.org#14397]
The following images are missing their alt text:

* Homepage: 
  * Featured Card Component (Donate) image
* Meet Mozilla:
  * Hero Image
  * Mission 
  * All History Images
* Make Pillar
  * Hero Image
* Inspire Pillar
  * Hero Image
  * Mozilla Stands for image
* Mobilize Pillar
  * Hero Image
  * Collective Action image
  * Investing image

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/TP1-3052)
